project/BuildPlugin.scala

```diff
@@ -415,7 +418,7 @@ object BuildImplementation {
      */
     def publishDocAndSourceArtifact(info: Option[GitDescribeOutput], version: String): Boolean = {
       val isStable = info.map(_.dirtySuffix.value.isEmpty)
-      !isStable.map(stable => !stable || version.endsWith("-SNAPSHOT")).getOrElse(false)
+      !isStable.exists(stable => !stable || version.endsWith("-SNAPSHOT"))
```

Comment:
couldn't resist here, sorry :) but i can revert this

Reply:
or change it to

Reply:
Let's keep it but let's keep changes focused :wink: